sdk/search/Azure.Search.Documents/tests/Utilities/SearchResources.cs

```diff
@@ -408,16 +409,11 @@ private async Task<SearchResources> CreateSearchServiceAndIndexAsync()
                 // Generate a random Index Name
                 IndexName = Settings.Random.GetName(8);
 
-                // Create a Track 1 client as a temporary work around until we
-                // have support for managing indexes in Track 2
-                Microsoft.Azure.Search.SearchServiceClient client =
-                    new Microsoft.Azure.Search.SearchServiceClient(
-                        SearchServiceName,
-                        new Microsoft.Azure.Search.SearchCredentials(PrimaryApiKey));
-                await client.Indexes.CreateWithHttpMessagesAsync(GetHotelIndex(IndexName));
+                SearchServiceClient client = new SearchServiceClient(Endpoint, new AzureKeyCredential(PrimaryApiKey));
+                await client.CreateIndexAsync(GetHotelIndex(IndexName));
 
                 // Give the index time to stabilize before running tests.
-                await Task.Delay(TimeSpan.FromSeconds(20));
+                await WaitForIndexingAsync();
```

Comment:
I don't think this is the same, is it?  These were tuned the hard way by Bruce and I'd love to keep them if there isn't a good reason to change them.

Reply:
There are other helps, but all that does is no-op if using a recording, or if a delay was specified. If we're merely playing back a test, we shouldn't waste time waiting. Maybe I'll just have a generic conditional waiter to avoid confusion about why we're waiting.

Reply:
We don't run this code path when playing back anything.

Reply:
Fair point, but 20 seconds? IMO, the service needs to use LROs in more places. This magic number is probably a safe bet, but means then it's too long in the typical case. All tests passed with the method I used, which, IIRC, was only 2 seconds. Do we really need to add another 18 seconds?

Reply:
Bruce said he spent a lot of time and energy getting this exact behavior dialed in and I'd prefer to stick with it if we can.

Reply:
I'll make it configurable. All 400+ live tests passed yesterday with only a 2 second delay. I'll default to 20 as noted here. /cc @brjohnstmsft